### PR TITLE
Added the new arrange icons

### PR DESCRIPTION
**Pull request by formalhaut69**
### Description of the Change
Updated the appearance of the Arrange icons.

### CODE REVIEW

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByGroupAction.java [hunk 1]
```diff
@@ -33,7 +33,9 @@
 import org.openide.util.NbBundle.Messages;
 
 @ActionID(category = "Arrange", id = "au.gov.asd.tac.constellation.plugins.arrangements.group.ArrangeByGroupAction")
-@ActionRegistration(displayName = "#CTL_ArrangeByGroupAction", surviveFocusChange = true)
+@ActionRegistration(displayName = "#CTL_ArrangeByGroupAction",
+        iconBase = "au/gov/asd/tac/constellation/plugins/arrangements/group/resources/arrangeByNodes.png",
```
Comment (cygnus-x-1) [inline CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByGroupAction.java:37]: Is this icon supposed to be the same as "Arrange by Node" icon?

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByNodeAttributeAction.java [hunk 2]
```diff
@@ -32,7 +32,9 @@
  * @author canis_majoris
  */
 @ActionID(category = "Arrange", id = "au.gov.asd.tac.constellation.plugins.arrangements.group.ArrangeByNodeAttributeAction")
-@ActionRegistration(displayName = "#CTL_ArrangeByNodeAttributeAction", surviveFocusChange = true)
+@ActionRegistration(displayName = "#CTL_ArrangeByNodeAttributeAction",
+        iconBase = "au/gov/asd/tac/constellation/plugins/arrangements/group/resources/arrangeByNodes.png",
```
Comment (cygnus-x-1) [inline CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByNodeAttributeAction.java:36]: arrangeByNode - singlular?

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByGroupAction.java (hunk 1 above)
Comment (formalhaut69) [reply CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByGroupAction.java:37]: Looks like that's a mistake, I don't know why I did that but I'll remove it.

File: CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByNodeAttributeAction.java (hunk 2 above)
Comment (formalhaut69) [reply CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByNodeAttributeAction.java:36]: Yeah, that should be arrangeByNodes I think. I'll change that

Comment (arcturus2) [reply CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/group/ArrangeByNodeAttributeAction.java:36]: @formalhaut69 just letting you know I haven't merged this PR as I was waiting on this change. No rush, whenever you can. Cheers.

**Pull request merged**